specification/deviceupdate/data-plane/Microsoft.DeviceUpdate/preview/2022-07-01-preview/deviceupdate.json

```
@@ -3427,23 +3450,23 @@
         },
         "createdDateTime": {
           "type": "string",
-          "description": "Date and time when the deviceclass subgroup was created."
+          "description": "Date and time when the device class subgroup was created."
         },
         "deviceCount": {
           "type": "integer",
-          "description": "The number of devices in the deviceclass subgroup."
+          "description": "The number of devices in the device class subgroup."
         },
         "deploymentId": {
           "type": "string",
-          "description": "The active deployment Id for the deviceclass subgroup."
+          "description": "The active deployment Id for the device class subgroup."
         }
       },
       "required": [
         "deviceClassId",
         "groupId",
         "createdDateTime"
       ],
-      "description": "Device class subgroup details."
+      "description": "Device class subgroup details. A device class subgroup is a subset of devices in a group that share the same device class id."
```

```suggestion
      "summary": "Device class subgroup details",
      "description": "A device class subgroup is a subset of devices in a group that share the same device class id."
```

This should help build better docstring

---

"Summary" is being ignored by autorest: https://github.com/Azure/autorest.csharp/issues/2388